addons/mail/controllers/discuss.py

```diff
@@ -285,3 +285,65 @@ def message_get_suggested_recipients(self, model, res_ids):
         except Exception:
             return {}
         return records._message_get_suggested_recipients()
+
+    # --------------------------------------------------------------------------
+    # RTC API
+    # --------------------------------------------------------------------------
+
+    @http.route('/mail/rtc/session/notify_call_members', type="json", auth="public")
+    def session_call_notify(self, sender, targets=None, content=None):
+        if request.env.user._is_public():
+            guest = request.env['mail.guest']._get_guest_from_request(request)
+            if guest:
+                guest.env['mail.channel.rtc.session'].sudo().browse(int(sender))._notify_peers(targets, content)
+                return
+        request.env['mail.channel.rtc.session'].sudo().browse(int(sender))._notify_peers(targets, content)
+
+    @http.route('/mail/rtc/session/update_and_broadcast', type="json", auth="public")
```

Comment:
Add `methods=['POST'],` to all routes that have side effects to avoid CSRF with GET, I'm not sure if it's actually necessary when type is json but better be safe.